apps/application-system/api/src/app/modules/application/application.module.ts

```diff
@@ -68,17 +66,13 @@ if (process.env.INIT_SCHEMA === 'true') {
   ],
   controllers: [ApplicationController],
   providers: [
-    ApplicationService,
     FileService,
     UploadProcessor,
     {
       provide: APPLICATION_CONFIG,
       useValue: environment.application as ApplicationConfig,
     },
-    AwsService,
     ApplicationAccessService,
-    ApplicationLifeCycleService,
   ],
-  exports: [ApplicationService, APPLICATION_CONFIG, AwsService],
 })
-export class ApplicationModule {}
+export class ApplicationsModule {}
```

Comment:
By renaming the `application/nest/application` library to `applicat/nest/core` and renaming the module to `CoreModule` would bypass the `ApplicationModule` vs `ApplicationsModule`